tests/test_porcelain.py

```diff
@@ -14,7 +14,7 @@ def test_list_all(tmpdir, runner, create):
     assert (
         result.output.strip() ==
         '{"completed": true, "due": 1451689200, "id": 1, "list": "default'
-        '", "percent": 26, "summary": "Do stuff", "urgent": false}'
+        '", "percent": 26, "summary": "Do stuff", "priority": None}'
```

Comment:
The output of `--porcelain` cannot lose any fields (it's part of the promise we keep, since other apps can integrate into it). We **can** add fields though.